multinet/api/management/commands/setupdevenv.py

```diff
@@ -0,0 +1,144 @@
+from arango.exceptions import (
+    ArangoClientError,
+    ArangoServerError,
+    PermissionUpdateError,
+    UserCreateError,
+)
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from multinet.api.utils.arango import arango_system_db
+from multinet.api.models import Workspace, Network, Table, TableTypeAnnotation
+from multinet.api.tasks.upload.csv import process_row
+
+from pathlib import Path
+from io import StringIO
+import csv
+import json
+
+
+
+class Command(BaseCommand):
+    help = "Sets up a standard development environment with example workspaces and tables."
+
+    def create_workspace_if_not_exists(self, workspace_name):
+        try:
+            # Check if workspace exists, if so remove it
+            if Workspace.objects.filter(name=workspace_name).exists():
+                self.stdout.write(self.style.SUCCESS(
+                    f"{workspace_name} already exists, removing to recreate"
+                ))
+                Workspace.objects.filter(name=workspace_name).delete()
+
+            # Create the workspace
+            Workspace.objects.create(name=workspace_name, public=True, owner=self.user)
+            self.stdout.write(self.style.SUCCESS(
+                f"{workspace_name} created"
+            ))
+
+        except Exception as e:
+            self.stderr.write(self.style.ERROR(str(e)))
+
+    def create_tables_for_workspace(self, workspace_name, edge_table_name):
+        try:
+            # Get the paths for all the data objects to upload
+            data_dir_path = Path(settings.BASE_DIR) / "data" / workspace_name
+            csv_paths = list(data_dir_path.glob("*.csv"))
+            workspace = Workspace.objects.get(name=workspace_name)
+
+            for csv_path in csv_paths:
+                filename = str(csv_path).split("/")[-1].split(".")[0]
```

Comment:
A more idiomatic way to do this

```suggestion
                filename = os.path.splitext(csv_path.name)[0]
```